gslab_scons/log.py

```diff
@@ -52,7 +53,7 @@ def end_log(log = 'sconstruct.log'):
             with open(f, 'rU') as sconscript:
                 if this_run_dict[f] == beginning_of_time:
                     sconstruct.write("*** Warning!!! Doesn't look the sconscript below finished.\n")
```

Comment:
Can you please do me a favor and change this one to " Warning!!! The log below does not have timestamps, the Sconscript may not have finished."